spec/elf/EverParse.h

```diff
@@ -0,0 +1,192 @@
+/*++
+
+Copyright (c) Microsoft Corporation
+
+Module Name:
+
+EverParse.h
+
+Abstract:
+
+This is an EverParse-generated file that contains common auxiliary
+functions for EverParse-generated verified data validators.
+
+This file was generated by EverParse v2022.02.02
```

Comment:
If this file was generated, can the generation process be part of the build rather than checking in the generated file?